doc/classes/TextEdit.xml

```diff
@@ -404,6 +404,7 @@
 			Adjust [code]Vertical[/code] Scroll Sensitivity of Text Editor.
```

Comment:
There's no reason to use Title Capitalization here. Also no reason to make "Vertical" inline code as it's not a code identifier. "Vertical scroll sensitivity." would be fine.